contracts/eth/nearprover/utils/upgrade_bridge_address.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+
+const { ethers } = require('hardhat');
+const { assert, expect } = require('chai');
+
+const BRIDGE_ADDRESS_SLOT = 2;
+
+async function upgradeProversBridgeAddressTo(provider, proverAddress, newBridgeAddress) {
+    const nearProverFactory = await ethers.getContractFactory('NearProver');
+    const nearProver = nearProverFactory.attach(proverAddress);
+
+    console.log(`Got prover at address: ${proverAddress}`);
+
+    const initialBridgeAddress = await nearProver.bridge();
+    console.log(`Initial bridge address: ${initialBridgeAddress}`);
+    console.log(`Trying to upgrade bridge address to: ${newBridgeAddress}`);
+
+    const adminWallet = new ethers.Wallet(process.env.ETH_PRIVATE_KEY, provider);
+
+    assert.equal(
+        adminWallet.address,
+        await nearProver.admin(),
+        "The used account is not an admin of NearProver"
+    );
+
+    // Mask matches only on the latest 20 bytes (to store the address)
+    const mask = ethers.BigNumber.from("0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff");
+    const response = await nearProver
+        .connect(adminWallet)
+        .adminSstoreWithMask(BRIDGE_ADDRESS_SLOT, newBridgeAddress, mask);
+    await response.wait().then(function(receipt) {
```

Comment:
Ethers by default only waits for 1 block, which is fine in most cases. In this sensitive case, let's use a bigger number (bigger than 5 at least).

```suggestion
    await response.wait(10).then(function(receipt) {
```